membership-attribute-service/app/wiring/AppLoader.scala

```diff
@@ -45,27 +45,14 @@ class MyComponents(context: Context)
     new AccountController(commonActions, controllerComponents)
   )
 
-  val regularCorsPaths = Seq(
-    "/user-attributes/me/membership",
-    "/user-attributes/me/features",
-    "/user-attributes/me"
-  )
-
-  val mmaPaths = Seq(
-    "/user-attributes/me/mma-digitalpack",
-    "/user-attributes/me/mma-monthlycontribution",
-    "/user-attributes/me/mma-membership",
-    "/user-attributes/me/mma-paper")
-
   val postPaths: List[String] = router.documentation.collect { case ("POST", path, _) => path }
 
   override lazy val httpFilters: Seq[EssentialFilter] = Seq(
     new CheckCacheHeadersFilter(),
     csrfFilter,
     new AddEC2InstanceHeader(wsClient),
     new AddGuIdentityHeaders(),
-    CORSFilter(corsConfig = Config.mmaCorsConfig, pathPrefixes = mmaPaths),
     CORSFilter(corsConfig = Config.mmaUpdateCorsConfig, pathPrefixes = postPaths),
-    CORSFilter(corsConfig = Config.corsConfig, pathPrefixes = regularCorsPaths)
```

Comment:
Previously, this was ONLY matching on "/user-attributes/me" anyway, so all the other paths were redundant.